integration/query_projection_test.go

```diff
@@ -60,7 +60,7 @@ func TestQueryProjectionErrors(t *testing.T) {
 		},
 		"ExcludeInclude": {
 			filter:     bson.D{},
-			projection: bson.D{{"foo", false}, {"bar", true}},
+			projection: bson.D{{"foo", false}, {"_id", true}, {"bar", true}},
```

Comment:
This test case is checking error message for mixed usage of inclusion and exclusion, so it's not our interested. We are interested in testing happy path instead of error case :hugs: 

Instead of changing `func TestQueryProjectionErrors` which tests error messages, let's write a test that asserts projection
```projection: bson.D{{"v", true}, {"_id", false}}``` gives expected result

Reply:
Let me know if you need any help :hugs: